tests/sklearn/test_sklearn_model_export.py

```diff
@@ -121,6 +121,7 @@ def sklearn_custom_env(tmp_path):
 
 
 def test_model_save_load(sklearn_knn_model, model_path):
+    assert False
```

Comment:
test, will revert

Reply:
worked as expected: https://github.com/mlflow/mlflow/actions/runs/6836235927/job/18590929638?pr=10361

Reply:
Revert this?

Reply:
reverted